gr-uhd/apps/uhd_app.py

```diff
@@ -325,7 +325,7 @@ def set_freq(self, freq, skip_sync=False):
                 treq = uhd.tune_request(
                     target_freq=freq,
                     rf_freq=freq,
-                    rf_freq_policy=uhd.tune_reqest.POLICY_MANUAL,
```

Comment:
how did this ever work?

Reply:
I think this is just an example of some code nobody ever uses. The variable `tune_reqest` does not appear within the UHD code base, and it was similarly misspelled in a couple other places (local variables that were consistently misspelled thus not causing any real problems) that this PR fixes.